responders/src/API/EMBC.Responders.API/Controllers/ProfileController.cs

```diff
@@ -166,7 +166,8 @@ public class UserProfile
         public string TeamId { get; set; }
         public string TeamName { get; set; }
         public string Role { get; set; }
-        public bool RequiredToSignAgreement { get; set; }
```

Comment:
I think this is used by the front end to determine the next route after log in

Reply:
Sue requested the change it to a date instead of a boolean.

Reply:
I'm pretty certain this change will break https://github.com/bcgov/embc-ess-mod/blob/c57b425f1bcfc9ecfe17adca0f499ce6da4086db/responders/src/UI/embc-responder/src/app/app.component.ts#L38